politeiawww/cmd/piwww/piwww.go

```diff
@@ -45,62 +45,67 @@ type piwww struct {
-	VoteInventory   VoteInventoryCmd   `command:"voteinventory" description:"(public) retrieve the tokens of all public, non-abandoned proposal separated by vote status"`
-
-	// Commands
-	ChangePassword     shared.ChangePasswordCmd `command:"changepassword" description:"(user)   change the password for the logged in user"`
-	ChangeUsername     shared.ChangeUsernameCmd `command:"changeusername" description:"(user)   change the username for the logged in user"`
-	EditUser           EditUserCmd              `command:"edituser" description:"(user)   edit the  preferences of the logged in user"`
-	Help               HelpCmd                  `command:"help" description:"         print a detailed help message for a specific command"`
+	VoteAuthorize   voteAuthorizeCmd   `command:"voteauthorize" description:"(user) authorize a proposal vote (must be proposal author)"`
+	VoteStart       voteStartCmd       `command:"votestart" description:"(admin) start the voting period on a proposal"`
+	VoteStartRunoff voteStartRunoffCmd `command:"votestartrunoff" description:"(admin)  start a runoff using the submissions to an RFP"`
+	VoteBallot      voteBallotCmd      `command:"voteballot" description:"(public) cast ballot of votes for a proposal"`
+	Votes           votesCmd           `command:"votes" description:"(public) get the vote tally for a proposal"`
+	VoteResults     voteResultsCmd     `command:"voteresults" description:"(public) get vote results for a proposal"`
+	VoteSummaries   voteSummariesCmd   `command:"votesummaries" description:"(public) retrieve the vote summary for a set of proposals"`
+	VoteInventory   voteInventoryCmd   `command:"voteinventory" description:"(public) retrieve the tokens of all public, non-abandoned proposal separated by vote status"`
+	// XXX www vote routes
+	VoteDetails  voteDetailsCmd  `command:"votedetails" description:"(public) get the details for a proposal vote"`
+	VoteStatus   voteStatusCmd   `command:"votestatus" description:"(public) get the vote status of a proposal"`
+	VoteStatuses voteStatusesCmd `command:"votestatuses" description:"(public) get the vote status for all public proposals"`
+
+	// User commands
+	UserDetails        userDetailsCmd          `command:"userdetails" description:"(public) get the details of a user profile"`
+	UserEdit           userEditCmd             `command:"useredit" description:"(user) edit the preferences of the logged in user"`
+	UserNew            userNewCmd              `command:"usernew" description:"(public) create a new user"`
+	UserPaymentsRescan userPaymentsRescanCmd   `command:"userpaymentsrescan" description:"(admin) rescan a user's payments to check for missed payments"`
+	UserPendingPayment userPendingPaymentCmd   `command:"userpendingpayment" description:"(user) get details for a pending payment for the logged in user"`
+	UserProposals      userProposalsCmd        `command:"userproposals" description:"(public) get all proposals submitted by a specific user"`
```

Comment:
The www `UserProposals` command is being deprecated.  This command will be deleted.  The pi proposals route will eventually allow filtering by user ID, but not for the intial release.